src/js/pannellum.js

```diff
@@ -478,6 +478,12 @@ function onDocumentMouseMove(event) {
         var pitch = ((Math.atan(event.clientY / renderer.canvas.height * 2 - 1) - Math.atan(onPointerDownPointerY / renderer.canvas.height * 2 - 1)) * 180 / Math.PI * vfov / 90) + onPointerDownPitch;
         pitchSpeed = (pitch - config.pitch) * 0.2;
         config.pitch = pitch;
+
+        var verbose = config.verbose || false;
+        if(verbose){
+            console.log("Yaw = ", yaw*-1);
```

Comment:
Hehe, good question, I'm not sure why either, but if I take the original yaw error it displays the point on a different position. Is it that possible?